ProjFS.Mac/PrjFSKext/VirtualizationRoots.cpp

```diff
@@ -553,35 +555,32 @@ void ActiveProvider_Disconnect(VirtualizationRootHandle rootIndex, PrjFSProvider
     RWLock_ReleaseShared(s_virtualizationRootsLock);
 }
 
-errno_t ActiveProvider_SendMessage(VirtualizationRootHandle rootIndex, const Message message)
+static PrjFSProviderUserClient* GetRetainedProvider(VirtualizationRootHandle rootHandle)
```

Comment:
I've been following the convention of using "index" when it's known to be a real virtualisation root index (>= 0) or "handle" when it might be one of the "special" handle values. (`RootHandle_None` etc.)